PurchasesTests/Mocks/MockRequestFetcher.swift

```diff
@@ -3,10 +3,12 @@
 // Copyright (c) 2020 Purchases. All rights reserved.
 //
 
-class MockRequestFetcher: RCStoreKitRequestFetcher {
+@testable import PurchasesCoreSwift
+
+class MockRequestFetcher: StoreKitRequestFetcher {
     var refreshReceiptCalled = false
 
-    override func fetchReceiptData(_ completion: @escaping RCFetchReceiptCompletionHandler) {
```

Comment:
I killed the `typedef` because I didn't feel it was adding much, but I can bring it back if others liked it